sdk/keyvault/test-resources.json

```diff
@@ -206,53 +206,53 @@
             "objectId": "[parameters('testApplicationOid')]",
```

Comment:
This is not part of the PR. Please undo this.

Reply:
This file has been rebased to the main branch version